websockets/py35/py35_test_client_server.py

```diff
@@ -3,9 +3,9 @@
 
```

Comment:
I actually did this in a first (unpublished) patch but the new name (with the `test_` prefix) got picked up by recursive test launchers such as `python -m unittest` or `nosetests` and would

1. fail on py34,
2. not actually test anything on py35 (since the class does not inherit from `unittest.TestCase`).

Given that the content of this module is actually included by `websockets/test_client_server.py` and executed there, I assumed that's how you wanted it to behave. Perhaps you have another name in mind to avoid the unsavory `py35/py35_` name? Maybe just a leading `_`?